CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via **issue**, email, or any other method with the owners of this repository before making a change.
+
+Please note and visit our [Code of Conduct here!](https://github.com/tnfdesigns/themeanager/blob/d3213d4c7a2bbe7dec607c8562b768e1c540f061/CODE_OF_CONDUCT.md "See Our Code of Conduct"). Be sure to follow it in all your interactions with the project.
+
+## Pull Request Process:
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+
+## New Issue Creation Process:
+
+1. Ensure that the issue is not already created.
+2. Create a new issue with the appropriate template.
+3. Verify that the issue is not already assigned to someone else.
```

Comment:
This seems like something that should be in claiming an existing issue. I would create a new section that mentions that

```suggestion

## Claiming an Issue
1. Verify that the issue is not already assigned to someone else.
```

Reply:
added this section and expanded your traing of thought, I believe.